rust/hyperlane-base/src/settings/chains.rs

```diff
@@ -175,11 +175,7 @@ impl ChainConf {
         let ctx = "Building merkle tree hook";
         // TODO: if the merkle tree hook is set for sealevel, it's still a mailbox program
```

Comment:
is this TODO removable now?

Reply:
the remark is still relevant but I the config is correct now so I'll remove